Route.ts

```diff
@@ -70,6 +70,19 @@ function typecastVariable(variable: string, variableDefinition: IOperationVariab
   }
 }
 
+function getDefaultTransformResponse(): AxiosTransformer[] {
+  const { transformResponse } = axios.defaults;
+ 
+  if (!transformResponse) {
+    return [];
```

Comment:
This should always exist right? Granted transformResponse is included in axios as a part of the defaults

Reply:
Or are the defaults being overridden?

Reply:
Yeah, it should technically exist as an AxiosTransformer[] seen here: https://github.com/axios/axios/blob/master/lib/defaults.js#L57-L65. 

I can't think of a case where it wouldn't be available, but Typescript was yelling at me because the types are `AxiosTransformer(a function) | AxiosTranformer[] | undefined`.

So I could remove `getDefaultTransformResponse` and do `this.transformResponseFn = axios.defaults.transformResponse as AxiosTransformer[]`. 

Does that look better? It seems like a lot of work just to use a function that we already have available here: https://github.com/Econify/graphql-rest-router/blob/master/Route.ts#L52-L58.

Reply:
I ended up using the axios.default transform always and type casting it because I can't think of a case where it wouldn't be available.